mysql/infoschema.go

```diff
@@ -301,9 +303,126 @@ func getConstraints(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]str
 			m[col] = append(m[col], constraint)
 		}
 	}
-	return primaryKeys, m, nil
+	// Get all the foreign keys for the table
+	refTables, _ := getRefTables(db, table)
```

Comment:
Need to handle error returned by getRefTables.
Same comment a few lines down where getForeignKey is called.

Reply:
Done